harness/determined/core/_tensorboard_mode.py

```diff
@@ -0,0 +1,14 @@
+import enum
+
+
+class TensorboardMode(enum.Enum):
```

Comment:
I prefer to leave it here. I tried putting it in _train.py but that lead to circular imports because `TensorboardMode` is used by `TrainContext`, `CheckpointContext`, and `_context.core.init()`, and importing it in `checkpoint.py` from `core._train` messes things up.